src/app/qgsmaptoolmovefeature.cpp

```diff
@@ -42,34 +42,55 @@ QgsMapToolMoveFeature::QgsMapToolMoveFeature( QgsMapCanvas *canvas, MoveMode mod
 
 QgsMapToolMoveFeature::~QgsMapToolMoveFeature()
 {
-  delete mRubberBand;
+  deleteRubberband();
 }
 
 void QgsMapToolMoveFeature::cadCanvasMoveEvent( QgsMapMouseEvent *e )
 {
   if ( mRubberBand )
   {
-    const QgsPointXY pointCanvasCoords = e->mapPoint();
-    const double offsetX = pointCanvasCoords.x() - mStartPointMapCoords.x();
-    const double offsetY = pointCanvasCoords.y() - mStartPointMapCoords.y();
-    mRubberBand->setTranslationOffset( offsetX, offsetY );
-    mRubberBand->updatePosition();
-    mRubberBand->update();
```

Comment:
why are these 2 lines dropped?

Reply:
If I'm not mistaken, `setTranslationOffset` already updates the rubburband, making the call to `updatePosition` and `update` redundant.